tests/mainnet/history/headers_with_proof/block_proofs_deneb/beacon_block_proof-22162263.yaml

```diff
@@ -0,0 +1,32 @@
+execution_block_header:
+  "0xfbf884a87d9b41c39363242970cea015afbc9b5ba6ab1ed34f407b2621987353"
+execution_block_proof:
+  - "0x0782512b0075cdc017383c74639c9dfe6b14a5a4244c2683fef4a491ecc0a730"
+  - "0xa91e24d4b51f23b5dc6ae1e27342533fa7dbf9394ce4808db64ad37a063c6267"
+  - "0x8553b496b975dcb747c0e6e496607c41e658f309ed8307bdd145a494c946f398"
+  - "0x19f4868e7b08709b8165dab5fa3fdd28c955217619aa081829eec5395a235825"
+  - "0xe830a8772ce297251a628f267934b21a7bd94b6e6415136b6518b5536ae877dc"
+  - "0xc2399738abb3bed5d2d2b39f9c6d732f6492d62c4d4669ea3d76a7e46410e948"
+  - "0x1dd63e221528163153337217e41d81e8afe876fecf9e20e40e5e743f48197981"
+  - "0xdb56114e00fdd4c1f85c892bf35ac9a89289aaecb1ebd0a96cde606a748b5d71"
+  - "0xc3a54c84046c5b2644325e0171426559df7fbc0624efa9820e7866a04ba1ad85"
+  - "0x0000000000000000000000000000000000000000000000000000000000000000"
+  - "0xf5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb4b"
+  - "0xe55723512d586a3fc7c83dfd06cafe74472a91428fa1fa59219e487bec6447ae"
+beacon_block_root:
+  "0x452eb0df043bab8ec645217e50a04ff6bc18ba9898468643e44a186b97bb94f1"
+beacon_block_proof:
+  - "0x663c1c7324b757120ded349256377339721f588378e544a35acc524df1edcf7f"
+  - "0x8b7a8a7628be95fcddb60ee03e9cae87ec3d4f0879b0b1572739ea8ab7d8b53a"
+  - "0x60cf79e426d83e9e9b3a69fa37013061d436769f633b0b13f3e7ebe3a659abfd"
+  - "0xc814f1c66c38f5f71377d617ebcab46ac2072e25b7a6df008fc9fcd2f2397b73"
+  - "0x9cd4ed4355d25562c47b862d56c4a72a5a1556bb5b1e5f264b5b8146cf6d2fe4"
+  - "0x606a5a8c25047dbcbc49046a942d06bd26410acfb9003aa61bc3a3b397ef640d"
+  - "0xc6acd6acb6dd81c816fd29c0f0ec699b8821d51b93beb9bfabc8b01329472ec6"
+  - "0x8daf2e3a815f8857ec56773400f06155db31de3eb23a9aebebe6a98a08346fdb"
+  - "0x0c44b033f3788a949910463d4418e4fbe805bcaf2d38bf717e9d36aa12b6f424"
+  - "0xc0ec6d13181f60d5d9aba44056b484a142a41cd327f30199e5d579dd63c371db"
+  - "0x9b0c9d881805365cbb27ceab374e9bae594290743a9aba318a67f0f82abcda27"
+  - "0xdd7a8e2fbbe4df2730c37e3026b4161b28cbb4a4586cec606fcaa61389969d63"
+  - "0x9f9ff19845ba987d672380b1a010edae98df5b3416f9b84e2d7aa3827a41f1c3"
+slot: 11378687
```

Comment:
I did verify this version, as I can parse the proof into a SSZ `Vector`.

Side note: The previous version of test vectors for this (capella) included also the historical summaries. I know that we can get them from the provided state but I think that I will add those still later on so that I can keep my testing flow the same. Either that or move all to using the beacon_state.ssz